variables.tf

```diff
@@ -264,6 +276,30 @@ variable "session_cloudwatch_log_group_name" {
   type        = string
 }
 
+variable "ssm_key_artifact_export_access_key_id" {
+  type        = string
+  description = "The AWS SSM Parameter Store parameter that contains the AWS access key of the user that can write to the assessment artifact export bucket (e.g. \"/assessment_artifact_export/access_key_id\")."
```

Comment:
I'm not sure if it's worth the bother but should these new variables consistently reference `IAM user` (as opposed to just `user`) as is done in the `ssm_key_artifact_export_region` variable?

Reply:
Agreed - no bother at all.  Please see https://github.com/cisagov/cool-assessment-terraform/pull/213/commits/9b2edb863f62cf11d0c94ce0dba98c3f65c16a4d.